Automattic/wp-calypso
# Site Profiler: Open get report form when clicking the Metrics CTA

Fixes https://github.com/Automattic/dotcom-forge/issues/6909

## Proposed Changes

* Adds functionality to open the Get Report form when clicking the menu CTA
* Adds functionality to close the form when clicking on the `X` icon
* The arrow icon in the top-right of the form has been replaced by a `X` icon to indicate closing
* The form is fixed positioned at the bottom of the viewport

![CleanShot 2024-05-10 at 18 27 17@2x](https://github.com/Automattic/wp-calypso/assets/3519124/a975efcc-30e8-497e-beb3-875cc46b15b2)

## Testing Instructions



* Apply this patch to your environment
* Navigate to the site-profiler and analyze a website
* Click on the main CTA button in the basic metrics menu 
![CleanShot 2024-05-10 at 18 23 36@2x](https://github.com/Automattic/wp-calypso/assets/3519124/d42a2d2a-62e4-4ebe-9d29-2323439ae49d)
* Check the form is displayed at the bottom and that you can scroll through the results
* Click on the `X` icon in the form
* Check the form is closed as expected
* Check the style and behavior is similar to the production version




## Pre-merge Checklist



~~- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?~~
~~- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?~~
~~- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?~~
